src/mapgen.cpp

```diff
@@ -13947,6 +13947,9 @@ void mx_fumarole(map &m, const tripoint &)
     std::vector<point> fumarole = line_to(x1, y1, x2, y2, 0);
     for (auto &i : fumarole) {
         m.ter_set(i.x, i.y, t_lava);
+        if (one_in(10)){
+         m.spawn_item(i.x-1, i.y-1, "chunk_sulfur");
```

Comment:
Code quality: spacing (`if( x ) {` etc., not `if (x){`)

This could use a random position roll instead of always putting it in the same relative spot.